packages/theming/theme-types/src/Color.types.ts

```diff
@@ -11,7 +11,10 @@
  *
  * `{semantic: "windowBackgroundColor"}`
  */
-export type ColorValue = string;
+
+import { ColorValue } from 'react-native';
+
+export { ColorValue } from 'react-native';
```

Comment:
So that's how you do that :D 
Repeating again that we may just want to get rid of the type since it's redundant.

Reply:
I responded to the other comment on why I'm hesitant to just remove this. Feel free to reactivate if you feel differently.